src/main/resources/org/jfrog/hudson/pipeline/declarative/steps/gradle/GradleStep/help-tool.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Tool name from Jenkins configuration.
```

Comment:
```suggestion
    Tool name from Jenkins configuration that is configured on Manage Jenkins » Global Tool Configuration
```
Would be great to let the user know where to find the tool name. I am not sure on where the tool is configured, can you verify that it's in Global tool please ?

Reply:
@aHenryJard
Yes, that's right.
Thanks 👍